src/loader.js

```diff
@@ -1,5 +1,5 @@
 import Embed from './components/embed'
-import URL from './components/utils'
+import { URL } from './components/utils'
```

Comment:
I don't see changes to utils in this PR. Does this fix something?

Reply:
Yes, webpack was complaining about non existing default export from utils